### DISCUSSION TITLE
Code in PR 43662 got removed in 2.10.3

### DISCUSSION THREAD

**Discussion by nicolasge** [apache/airflow#44946] [General]
### Apache Airflow version

2.10.3

### What happened?

One issue was fixed in PR https://github.com/apache/airflow/pull/43662 came up again.  The codes to fix the issue was removed somehow****

### What you think should happen instead?

code in PR 43662 should persist or Airflow has another way to fix the issue in this PR

### How to reproduce

Pls refer to https://github.com/apache/airflow/pull/43662

### Operating System

Debian 12.7

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
This code is in FAB provider, not in airflow. please refer to changelog there - and see if you have the right provider installed https://airflow.apache.org/docs/apache-airflow-providers-fab/stable/index.html - look at which FAB provider version you have installed 

If you still have some kind of error - related to it - please open a new issue describing your problem and circumstances around it. 

You can also - if you are not sure - check the 2.10.4rc1 release that is just being voted (see https://github.com/apache/airflow/issues/44811) - and look up the version 2.10.4 rc1 has.

**Comment by potiuk** [comment]
Converting to a discussion if more is needed.
